examples/crowdfunding/mandos/crowdfunding-fund-too-late.scen.json

```diff
@@ -0,0 +1,77 @@
+{
+    "name": "trying to fund one block too late",
+    "steps": [
+      {
+        "step": "externalSteps",
+        "path": "crowdfunding-fund.scen.json"
+      },
+      {
+        "step": "setState",
+        "currentBlockInfo": {
+            "blockNonce": "123,001"
+        }
+      },
+      {
+        "step": "scCall",
+        "txId": "fund-too-late",
+        "tx": {
+            "from": "''donor_1_________________________",
+            "to": "''the_____crowdfunding____contract",
+            "value": "10,000,000,000",
+            "function": "fund",
+            "arguments": [],
+            "gasLimit": "100,000,000",
+            "gasPrice": "0"
+        },
+        "expect": {
+            "status": "4",
+            "message": "str:cannot fund after deadline",
+            "gas": "*",
+            "refund": "*"
+        }
+      },
+      {
+        "step": "checkState",
+        "accounts": {
+          "''my___personal__account___address": {
```

Comment:
same here with the address, a mass replace should do
also drop some of the underscores, "address:my_address"